### PR TITLE
Milestone 3 - RWD with Bootstrap

### PR DESCRIPTION
**Pull request by omar-labana**
In the previous milestone we:

- Created the HTML skeleton for the header and footer, and some basic styling with SASS.
- Added Bootstrap to the project and applied its classes to position the tags.
- Prepared the required assets from the original web page.
- Applied different positioning techniques using Flex box,
- Applied styling on the different web page components to look similar to the original web page.
- Created the HTML skeleton main section of the website for the 3 screen sizes, and some basic styling with bootstrap classes.
- Prepared some of the required assets from the original web page.
- Applied different positioning techniques using bootstrap grid system,
- Applied styling on the different web page components to look similar to the original web page.

In this mile stone we
- Added in the magazine, Subscribe / Newsletter, and Editor’s pick HTML skeleton.
- Did the styling and positioning of the mentioned section.

### CODE REVIEW

**Review by mricanho — COMMENTED** [review]
# 🔴 Invalid Submission 🔴  

Hello @omar-labana @enionsouza, unfortunately, I need to invalid this submission, you missed two main sections:

- Newspaper main sections
- Subscribe / Newsletter
shine team


Ask for a new pull request once you finish this milestone.

You can contact me on slack if you have any questions just search for me as @ Miguel Ricaño.

### 💻 👩🏽‍💻  Happy Coding!👨🏽‍💻  🖥️

File: .github/workflows/linters.yml [hunk 1]
```diff
@@ -45,4 +45,4 @@ jobs:
           npm install --save-dev stylelint@13.3.x stylelint-scss@3.17.x stylelint-config-standard@20.0.x stylelint-csstree-validator
           [ -f .stylelintrc.json ] || wget https://raw.githubusercontent.com/microverseinc/linters-config/master/html-css/.stylelintrc.json
       - name: Stylelint Report
-        run: npx stylelint "**/*.{css,scss}"
+        run: npx stylelint "**/*.scss"
```
Comment (sinansevgi) [inline .github/workflows/linters.yml:48]:
```suggestion
      run: npx stylelint "**/*.{css,scss}"
```

**Review by sinansevgi — COMMENTED** [review]
This comment edited for preventing any confusion. Please ignore this

File: .github/workflows/linters.yml (hunk 1 above)
Comment (omar-labana) [inline .github/workflows/linters.yml:48]: It was mentioned in the linter config that we do this
+ we are only using sass

Comment (omar-labana) [reply .github/workflows/linters.yml:48]: link https://github.com/microverseinc/linters-config/tree/master/html-css#troubleshooting

Comment (sinansevgi) [reply .github/workflows/linters.yml:48]: Hello, I am so sorry for the inconvenience.  
Please submit another review.

**Review by sinansevgi — CHANGES_REQUESTED** [review]
# Project status: Approved

----
Excellent job!. 

Between all the projects that I reviewed for this section of the curriculum. You are the only team that uses sass for their project. Kudos for this.

You can fix these three small issues so your project can look more professional

- [x] Image sizes in the ` In The Magazine` section are inconsistent and brokes your design. 

Screenshot_2021-04-09 News Week Clone


- [x] All the images in the ` In The Magazine`  section should have links like other images.
- [x] Size of newsletter form bigger than it should be and brokes your design. 
Screenshot_2021-04-09 News Week Clone(2)
- [x] Input field of newsletter form doesn't get into the correct order in mobile view.

| Original                                                                                                                                | Yours                                                                                                                                             |
|-----------------------------------------------------------------------------------------------------------------------------------------|---------------------------------------------------------------------------------------------------------------------------------------------------|
| Screenshot_2021-04-09 Newsweek | Screenshot_2021-04-09 News Week Clone(3) |

----
Sinan Sevgi

**Comment by enionsouza** [comment]
All issues raised by our code reviewer have been corrected.

**Pull request closed without merging**